.github/workflows/advanced.yml

```diff
@@ -148,23 +149,34 @@ jobs:
         run: |
           .github/scripts/build-aarch64-tests.sh
 
+      - name: Pack tests
+        run: |
+          tar czf ${{ env.TESTS_PACKAGE_NAME }} -C tests/build/bin/ .
```

Comment:
I'd move this code into a seperate bash script for better testing.

Reply:
This PR is getting large. Will follow-up in next PR to avoid merging issues.